decidim-core/config/locales/ca.yml

```diff
@@ -538,32 +564,7 @@ ca:
         close_modal: Tanca el modal
         filter: Filtre
         filter_by: Filtra per
-        unfold: Desplegar
```

Comment:
I don't think this key movement is intended.

Reply:
Thanks! I catched it in the spanish translation but missed it here.